frontend/prisma/migrations/20240708094810_/migration.sql

```diff
@@ -1,23 +1,24 @@
 /*
-  Warnings:
+Warnings:
 
-  - You are about to drop the `RVSP` table. If the table is not empty, all the data it contains will be lost.
+- You are about to drop the `RSVP` table. If the table is not empty, all the data it contains will be lost.
 
-*/
+ */
 -- DropForeignKey
-ALTER TABLE "RVSP" DROP CONSTRAINT "RVSP_eid_fkey";
+ALTER TABLE "RSVP"
+DROP CONSTRAINT "RSVP_eid_fkey";
 
 -- DropTable
-DROP TABLE "RVSP";
+DROP TABLE "RSVP";
 
 -- CreateTable
-CREATE TABLE "RSVP" (
+CREATE TABLE
```

Comment:
can u delete all the migrations files and then run `npx prisma migrate dev`, thatll clean things up here